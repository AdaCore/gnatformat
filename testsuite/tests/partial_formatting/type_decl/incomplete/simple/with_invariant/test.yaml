description: IncompleteTypeDecl formatting for line selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 73",
    "-EL 74",
  ]
status_code: 1