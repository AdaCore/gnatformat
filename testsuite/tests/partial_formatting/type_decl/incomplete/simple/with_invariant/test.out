<CompilationUnit test.ads:27:1-149:23>
test.ads(<IncompleteTypeDecl ["Person"] test.ads:73:4-73:20>) - 73:1-73:20
^
   type Person (<>);$

